<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="32984" started-at="2021-04-01T13:42:55Z" finished-at="2021-04-01T13:43:28Z">
    <groups>
      <group name="IntegrationTests">
        <method signature="Restassured_Automation_MethodologItem.MethodologyItem_DeleteMarkAnItemAsDeleted_status200()[pri:0, instance:restassured.automation.testcases.Restassured_Automation_MethodologItem@1554909b]" name="MethodologyItem_DeleteMarkAnItemAsDeleted_status200" class="restassured.automation.testcases.Restassured_Automation_MethodologItem"/>
      </group> <!-- IntegrationTests -->
    </groups>
    <test name="Default test" duration-ms="32984" started-at="2021-04-01T13:42:55Z" finished-at="2021-04-01T13:43:28Z">
      <class name="restassured.automation.testcases.Restassured_Automation_MethodologItem">
        <test-method status="PASS" signature="setup()[pri:0, instance:restassured.automation.testcases.Restassured_Automation_MethodologItem@1554909b]" name="setup" is-config="true" duration-ms="11" started-at="2021-04-01T19:12:55Z" finished-at="2021-04-01T19:12:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="MethodologyItem_DeleteMarkAnItemAsDeleted_status200()[pri:0, instance:restassured.automation.testcases.Restassured_Automation_MethodologItem@1554909b]" name="MethodologyItem_DeleteMarkAnItemAsDeleted_status200" duration-ms="32962" started-at="2021-04-01T19:12:55Z" finished-at="2021-04-01T19:13:28Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at restassured.automation.listeners.ExtentTestManager.statusLogMessage(ExtentTestManager.java:37)
	at restassured.automation.testcases.Restassured_Automation_MethodologItem.MethodologyItem_DeleteMarkAnItemAsDeleted_status200(Restassured_Automation_MethodologItem.java:1061)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MethodologyItem_DeleteMarkAnItemAsDeleted_status200 -->
      </class> <!-- restassured.automation.testcases.Restassured_Automation_MethodologItem -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
